bin/micro.js

```diff
@@ -16,12 +16,10 @@ const logError = require('../lib/error')
 
 // Check if the user defined any options
 const flags = parseArgs(process.argv.slice(2), {
-  default: {
-    port: 3000
-  },
   alias: {
     p: 'port',
     H: 'host',
+    s: 'socket',
```

Comment:
We might call it `unix-socket` to make it clear? which is `curl`'s option name.

Reply:
👍 good call